Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h

```diff
@@ -74,12 +74,41 @@ DECLARE_SOA_COLUMN(TOFNSigmaHe, tofNSigmaHe, float);
 DECLARE_SOA_COLUMN(TOFNSigmaAl, tofNSigmaAl, float);
 } // namespace pidtof
 
+namespace pidtof_tiny
+{
+constexpr int nbins = 256;
```

Comment:
is there a constexpr sizeof? then we could derive it from int8_t, or make even int8_t a define here

Reply:
uhm that's a good point !